gene_code_utils/dna_rna_tools_utils.py

```diff
@@ -0,0 +1,176 @@
+DNA = set ('ATGCatgc')
+RNA = set ('AUGCaugc')
```

Comment:
Здорово, что учла комметарий от Никиты. 
Есть недочет: между функцией (в данном случае у тебя функция преобразования типа) и аргументом пробел не ставится.
```suggestion
DNA = set('ATGCatgc')
RNA = set('AUGCaugc')
```